docs-site/docs/configuration/authentication.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 1
+id: authentication
+title: Connecting to an Authentication Server
+---
+
+## Setup
+
+### Enable remote authentication
+
+By default, remote authentication is disabled. To enable it you will need to edit the config file `config/auth_config.yml`
+
+It looks like this.
```

Comment:
Can you update this to `The auth_config.yml has the format below`

Reply:
fixed.